acbs/utils.py

```diff
@@ -326,3 +326,12 @@ def format(self, record):
                               logging.INFO, logging.DEBUG):
             record.msg = f'[{lvl_map[record.levelname]}]: \033[1m{record.msg}\033[0m'
         return super(ACBSLogFormatter, self).format(record)
+
+def get_defines_file_path(location, stage2=False) -> str:
+    '''
+    Return ${location}/defines or ${location}/defines.stage2 on various conditions.
+    '''
+    if stage2 == False:
```

Comment:
Use truthy comparison here:

```suggestion
    if not stage2:
```

We use typed Python in this project, so `stage2` must be a boolean type here.